src/python/pants/backend/jvm/tasks/junit_run.py

```diff
@@ -90,10 +89,13 @@ def register_options(cls, register, register_jvm_tool):
                   'For example, 1/3 means run tests number 2, 5, 8, 11, ...')
     register('--suppress-output', action='store_true', default=True,
              help='Redirect test output to files in .pants.d/test/junit.')
-    # TODO(gmalmquist): Use the build root instead of the path of the first target by default.
     register('--cwd', default=_CWD_NOT_PRESENT, nargs='?',
-             help='Set the working directory. If no argument is passed, use the first target path. '
+             help='Set the working directory. If no argument is passed, use the build root. '
                   'If cwd is set on a target, it will supersede this argument.')
+    register('--cwd-default-to-first-target', action='store_true',
```

Comment:
I added the behavior of `--cwd` to mean use the dir of the first target.  I don't think it is used outside of our repo, and I think setting `cwd` on the target is a better solution.  IntelliJ can inherit from it, for example.

My initial RB was greeted with skepticism but tolerance during the review, and has befuddled reviewers ever since.  I think it is time for it to go.